java/client/src/main/java/glide/ffi/resolvers/ScriptResolver.java

```diff
@@ -0,0 +1,25 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.ffi.resolvers;
+
+public class ScriptResolver {
+
+    // TODO: consider lazy loading the glide_rs library
+    static {
+        System.loadLibrary("glide_rs");
+    }
+
+    /**
+     * Loads a Lua script object to Redis
+     *
+     * @param code The Lua script
+     * @return String representing the saved hash
+     */
+    public static native String storeScript(String code);
+
+    /**
+     * Unload or drop the stored Lua script from Redis by hash
```

Comment:
The script doesn't actually get removed from the Redis cache.
We only drop it from our "local cache", since we locally save the script object instead of just loading it into the cache.

from Redis doc: The script is guaranteed to stay in the script cache **forever** (unless SCRIPT FLUSH is called).

Reply:
Updated